elastic/security/templates/composable/logs-endpoint.events.file.json

```diff
@@ -23,12 +19,10 @@
     "composed_of": [
       "logs@mappings",
       "logs@settings",
-      "ecs@mappings",
       "logs-endpoint.events.file@package",
-      "logs-endpoint.events.file@custom",
+      "ecs@mappings",
       ".fleet_globals-1",
-      ".fleet_agent_id_verification-1",
-      "track-shared-logsdb-mode"
```

Comment:
@salvatore-campagna Are you OK with removal of `track-shared-logsdb-mode` here? Today when index mode is `logsdb` we take composable template definition for `logs-endpoint` integration from `composable-logsdb` directory, not `composable` directory like this file here. Do we need `track-shared-logsdb-mode` component template in any of composable templates in `composable` directory?

Reply:
The assumption here is that `track-shared-logsdb-mode` is not required by others like `.logs-endpoint.action.responses` because for those we rely on kibana. I would then just remove it and remove it also from `composed_of`.

Reply:
Ok, so let's remove including `security-*` composable templates. Please also remove `track-shared-logsdb-mode` component template installation when not in `logsdb` index mode in `track.json`.

Reply:
Done